contact.php

```diff
@@ -1,33 +1,73 @@
+<?php
+ini_set('display_errors', 1);
+require_once('config.php');
+//Check for form submission - Is the form empty? 
+if (!empty($_POST)) {
+	//Connect and escape
+	//$mysql = new mysqli(DB_HOST, DB_USER, DB_PASS, DB_NAME);
+	//Submit to database
+	//$query = "INSERT INTO users (name,email) VALUES ('$name','$email')";
+	//$insert = $mysql->query($query);
+}
+?>
 <!DOCTYPE html>
 <html lang="en">
-    
-<head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"/>
-    
+<head>
+	<meta charset="utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous" />
 </head>
 
 <body>
-    <div name="form">
-        <form method="post" name="your-message">
-            Full Name: <input type="text" name="fullName"/><br>
-            Email: <input type="email" name="email"/><br>
-            Your Message: <textarea name="yourMessage">type here</textarea><br>
-            <input type="submit" value="send message"/>
-        </form>
-    </div>
+	<div class="container">
+		<div name="form">
+			<?php
+			//Define variables set to empty values
+			$name = '';
+			$email = '';
+			$nameErr = '';
+			$emailErr = '';
 
-    <?php 
-        $fullName = $_POST["fullName"];
-        $email = $_POST["email"];
-        $yourMessage = $_POST["yourMessage"];
-
-        echo "Hello $fullName <br>";
-        echo "We will respond to your message @ $email <br>";
-        
-    ?>
-    
+			if (!empty($_POST['submit'])) {
```

Comment:
use isset rather than not empty since a button doesn't need a value but can still be "set" in `$_POST`